WordPress/openverse-frontend
# Simplify Filter Sidebar component and improve scrollbar styles

## Fixes


(slightly) Related to #412 by @panchovm

## Description


This PR removes one of the nested Sidebar Filter components to simplify it, and also removes the gap between the sidebar and main content by setting the scrollbar background to transparent. The scrollbar for Filter Sidebar is complicated with the iframe: it now technically doesn't overflow, so the 'thumb' of the scrollbar is invisible, but the gap is still there.
This PR is based on the `VCheckbox` PR.

## Testing Instructions

Run the app and open the Filter sidebar. You should not see this gap between it and the content:
<img width="504" alt="Screen Shot 2021-11-18 at 3 53 59 PM" src="https://user-images.githubusercontent.com/15233243/142435751-839545c6-a716-4e39-904c-2325ad2009d2.png">

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
